<launch>
    <node pkg="a_star_test" type="a_star_test" name="a_star_test" output="screen">
        <param name="start_point_x" type="double" value="-2.821"/>
        <param name="start_point_y" type="double" value="1.483"/>
        <param name="start_point_z" type="double" value="0.0"/>
        <param name="angle" type="double" value="0.0"/>
    </node>
</launch>